### DISCUSSION TITLE
what is the meaning of "Requested performance error"?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#174] [Q&A]
![image](https://user-images.githubusercontent.com/5112397/149297813-0b9da473-bdd1-4a37-a5a7-69f83e0d1c19.png)

**Comment by ishwaran** [comment]
Requested performance error means the error rate determined by `perf.type`.  Many different types of error rates are reported, but we make a note of the one that is specifically requested by the user.
